File: RemixedDungeon/src/main/java/com/watabou/pixeldungeon/actors/buffs/SnipersMark.java

package com.watabou.pixeldungeon.actors.buffs;

import com.nyrds.LuaInterface;
import com.watabou.pixeldungeon.ui.BuffIndicator;

@LuaInterface
public class SnipersMark extends FlavourBuff {
	
	@Override
	public int icon() {
		return BuffIndicator.MARK;
	}
}

File: scripts/buffs/SnipersMark.lua
---
--- SnipersMark: marks a target for the sniper (was actors/buffs/SnipersMark.java)
---
local RPD  = require "scripts/lib/commonClasses"
local buff = require "scripts/lib/buff"

return buff.init{
    desc  = function ()
        return {
            icon          = 27, -- BuffIndicator.MARK
            name          = "SnipersMarkBuff_Name",
            info          = "SnipersMarkBuff_Info",
        }
    end
}
